public/data/ankushtripathii.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Ankush Tripathi",
+  "type": "personal",
+  "bio": "I am Ankush Tripathi, currently pursuing my engineering degree in computer science from Atharva college of engineering. I am a tech enthusiast. I am always open to collaborate on projects and innovative/disruptive ideas",
+  "avatar": "https://github.com/ankushtripathii.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/ankushtripathii",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/imalphaorionis",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Linkedin",
```

Comment:
I recommend capitalizing the `i` here. What do you think? :)
```suggestion
      "name": "Follow me on LinkedIn",
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inelegant

Reply:
> 

yup right thanks